docker/configFile.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
I suggest leaving it as it is for now and make an issue on JIRA about that to resolve it later.